Sources/Kontur/MessageSubscriber.cs

```
@@ -27,6 +31,17 @@ public ISubscriptionTag SubscribeTo(ISourceBlock<IMessage> target)
             return new SubscriptionTag(Guid.NewGuid().ToString(), this);
         }
 
+        private void OnMessage(IMessage message)
+        {
+            try
+            {
+                action(this.As(message));            
+            }
+            catch (System.Exception)
+            {

```

Empty catch? What should be done here?

---

I think this is a place of the implementation of https://github.com/mzabolotko/Kontur/issues/36 and https://github.com/mzabolotko/Kontur/issues/38